### DISCUSSION TITLE
The mail server for github.com is not accepting our messages to sherifrissas@github

### DISCUSSION THREAD

**Discussion by sherifrisas** [community/community#22111] [New to GitHub]
<p>the mail server for <a href="http://github.com" rel="nofollow noopener">github.com</a> is not accepting our messages to sherifrissas@github</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>GitHub accounts don’t include an <code>@github.com</code> email address. Only employees would have those!</p>
<p>If I’ve misunderstood your question, let me know!</p>
